README.md

```diff
@@ -1,16 +1,1261 @@
+      "do": "msg/send",
+      "inputs": {
+        "to": ["boris@example.com", "carol@example.com"],
+        "subject": "Coffee",
+        "body": "Hey you two, I'd love to get coffee sometime and talk about UCAN Tasks!"
+      },
+      "meta": {
+        "dev/tags": ["friends", "coffee"],
+        "dev/priority": "high",
+        "dev/notes": {
+          "select-task": ["/", "some-label"] // <- Pointer here
+        }
+      }
+    }
+  },
+  "prf": [{"/": "bafkreibbz5pksvfjyima4x4mduqpmvql2l4gh5afaj4ktmw6rwompxynx4"}],
+  "sig": {"/": {"bytes:": "5vNn4--uTeGk_vayyPuNTYJ71Yr2nWkc6AkTv1QPWSgetpsu8SHegWoDakPVTdxkWb6nhVKAz6JdpgnjABppC7"}}
+}
+```
+
+#### 7.2.2.1 List
+
+This absolute pointer:
+
+``` json
+[{"/": "bafkreiew2p74l7bq3hnllbduzagdcezlab54ko4lpw72mfcvilh4ov2hkq"}, "1"]
+```
+
+Will select the marked field in this List invocation:
+
+``` json
+// CID = bafkreiew2p74l7bq3hnllbduzagdcezlab54ko4lpw72mfcvilh4ov2hkq
+{
+  "uiv": "0.1.0",
+  "nnc": "6c*97-3=",
+  "run": [
+    {
+      "with": "mailto:akiko@example.com",
+      "do": "msg/send",
+      "inputs": {
+        "to": ["boris@example.com", "carol@example.com"],
+        "subject": "Coffee",
+        "body": "Hey you two, I'd love to get coffee sometime and talk about UCAN Tasks!"
+      },
+      "meta": {
+        "dev/tags": ["friends", "coffee"],
+        "dev/priority": "high",
+        "dev/notes": {
+          "select-task": ["/", "0"] // <- Pointer here
+        }
+      }
+    },
+    // Selects this
+    // vvvvvvvvvvvv
+    { 
+      "with": "https://example.com/blog/posts",
+      "do": "crud/create",
+      "inputs": {
+        "headers": {
+          "content-type": "application/json"
+        },
+        "payload": {
+          "title": "How UCAN Tasks Changed My Life",
+          "body": "This is the story of how one spec changed everything...",
+          "topics": ["authz", "journal"],
+          "draft": true
+        }
+      }
+    }
+  ],
+  "prf": [{"/": "bafkreibbz5pksvfjyima4x4mduqpmvql2l4gh5afaj4ktmw6rwompxynx4"}],
+  "sig": {"/": {"bytes:": "5vNn4--uTeGk_vayyPuNTYJ71Yr2nWkc6AkTv1QPWSgetpsu8SHegWoDakPVTdxkWb6nhVKAz6JdpgnjABppC7"}}
+}
+```
+
+# 8 Result
+
+A Result records the output of a [Task](#4-task), as well as its success or failure state.
+
+## 8.1 Variants
+
+``` ipldsch
+type Result union {
+  | Any ("ok")  -- Success
+  | Any ("err") -- Failure
+} representation keyed
+```
+
+### 8.1.1 Success
+
+The success branch MUST contain the value returned from a successful Task wrapped in the `"ok"` tag. The exact shape of the returned data is left undefined to allow for flexibility in various Task types.
+
+``` json
+{"ok": 42}
+```
+
+### 8.1.2 Failure
+
+The failure branch MAY contain detail about why execution failed wrapped in the `"err"` tag. It is left undefined in this specification to allow for Task types to standardize the data that makes sense in their contexts.
+
+If no information is available, this field SHOULD be set to `Null`.
```

Comment:
Can that instead implicitly be `{}` ? In my experience it's always a better idea than `Null` as that allows backwards compatible extensions in the future when `Null` no longer suffice.

I don't feel strongly about it, just would love spec to nudge users into right direction.

Reply:
Sure, that's reasonable 👍